src/wizard/wizard-dialog.ts

```diff
@@ -181,8 +181,8 @@ export class ESPHomeWizardDialog extends LitElement {
         label="Name"
         name="name"
         required
-        pattern="^[a-z0-9-]+$"
-        helper="Only use lowercase letters (a-z), numbers (0-9) or dash (-)"
+        pattern="^[a-z0-9-_]+$"
+        helper="Only use lowercase letters (a-z), numbers (0-9), dash (-) or underscore (_)"
         @input=${this._cleanNameInput}
```

Comment:
This method will still replace `_` with `-`.

Reply:
Sorry, missed that!